src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/PreferDictionaryTryGetValueMethodsTests.cs

```diff
@@ -0,0 +1,270 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
+using Microsoft.NetCore.Analyzers.Runtime;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
```

Comment:
Needs similar VB tests.

Reply:
Partially done. I'm having some problems with the VB fixer. As VB does not have `out` variables, I need to insert a variable before the condition. When I do this (line 45 of the fixer), I get the following exception: System.InvalidOperationException: The item specified is not the element of a list. Do you have any pointers?

Reply:
Seems it should be declaration not assignment ~~Dim value = Nothing~~ Dim value As Integer

Reply:
Thanks, this helped.